lms/envs/common.py

```diff
@@ -2321,6 +2321,9 @@ def _make_locale_paths(settings):
     'csrf.apps.CsrfAppConfig',  # Enables frontend apps to retrieve CSRF tokens.
 
     'edly_panel_app',
+
+    # q-verse app for registration
+    'openedx.features.qverse_features.registration.apps.RegistrationConfig'
```

Comment:
just curious, will it works if we change it to **'openedx.features.qverse_features.registration'**?

I think it should work as you have already set default_app_config in __init__.py

Reply:
done.